syntax = "proto3";
package Proto;

import "Exchange.proto";

enum InfoType
{
  INFO = 0;
  WARN = 1;
  ERROR = 2;
}

message ServerInfo
{
  Exchange exchange = 1;
  string info = 2;
  InfoType type = 3;
  uint64 time = 4;
}
